exasol_integration_test_docker_environment/cli/commands/spawn_test_environment.py

```diff
@@ -134,6 +142,7 @@ def spawn_test_environment(
                 environment_name,
                 database_port_forward,
                 bucketfs_port_forward,
```

Comment:
Same as with CLI, we should add bucketfs_http_port_forward and deprecate the old one.